### ISSUE TITLE
Allow `SetMenuOptions` to be used for the `enum` control type

### ISSUE THREAD

**Issue by qudix** [Exit-9B/MCM-Helper#8]
I have two arrays:
```papyrus
Actor[] Property ActorList Auto ; contains a list of actors
String[] Property ActorNameList Auto ; contains the names of those actors in the same order.
```
I then pass the name array to:
```papyrus
SetMenuOptions("...", ActorNameList)
```
At the moment there is no reliable way to map the selection to the actor array without iterating through each actor and comparing their names to the selection. This is also more problematic when two or more actors have the same name (rare, but possible). ~~I propose adding an option to the Menu control to return the selected index instead (using `sourceType` if possible would make sense).~~

Edit: The alternative proposal is now to allow SetMenuOptions on the 'enum' control instead.

**Comment by Exit-9B** [comment]
Next update we'll allow `SetMenuOptions` to be used for the enum control type as well.

**Comment by Exit-9B** [comment]
Resolved via 3ec2291.
